hs/src/Reach/Eval/Core.hs

```diff
@@ -2958,6 +2984,20 @@ evalPrim p sargs =
       ensure_after_first
       zero_args
       evalPrim (SLPrim_fluid_read FV_didSend) []
+    SLPrim_Unstrict -> do
+      one_arg >>= \case
+        SLV_Clo clo_at Nothing clo@(SLClo _ clo_args _ _) -> do
+          at <- withAt id
+          unless (null clo_args) $
+            expect_thrown clo_at $
+              Err_Apply_ArgCount at 0 (length clo_args)
```

Comment:
I don't think you should bother with this. What's the harm in allowing someone to write `unstrict((x = 1) => { ... })` or to see an application error if they write `unstrict((x) => { ... })`?